### ISSUE TITLE
Environment variables missing when running docker-compose

### ISSUE THREAD

**Issue by franck-fourel** [tidepool-org/tools#68]
I cannot run the full tidepool stack with the docker-compose since I updated all my images with docker-compose pull. Below is the list of containers that fail during start-up command

``` 
docker-compose up -d
``` 

![dockerstatus](https://user-images.githubusercontent.com/22294614/37028986-29e47770-2136-11e8-9ef6-bdcfe55256b6.PNG)

I noticed that the Dockerfiles of the failing images have been modified recently. For example the hakken dockerfile does not contain anymore the environment variables: https://github.com/tidepool-org/hakken/commit/f491a2983e0b27784f96ac63e0dd99f488ee1b23 

Below is the log for this hakken service:

```
Attaching to hakken
hakken            | /app/node_modules/amoeba/lib/config.js:22
hakken            |       throw except.ISE('Must specify environment variable[%s]', name);
hakken            |       ^
hakken            |
hakken            | Error: Must specify environment variable[ANNOUNCE_HOST]
hakken            |     at Object.ISE (/app/node_modules/amoeba/lib/except.js:24:18)
hakken            |     at Object.exports.fromEnvironment (/app/node_modules/amoeba/lib/config.js:22:20)
``` 

I'm wondering if I missed something or if my docker images were not up to date but all of the failing services have the same kind of error related to missing environment variables. 

I also noticed the variables are still available in https://github.com/tidepool-org/tools/blob/master/runservers#L136  
Would that be a regression only in docker-compose or something that I missed in documentation?

**Comment by franck-fourel** [comment]
Thanks a lot.
I will test it right now and I will tell you if I have any issue.


Le mar. 6 mars 2018 à 15:27, Clint Beacock <notifications@github.com> a
écrit :

**Comment by cbwebdevelopment** [comment]
Closed by #69
